jfrog-tasks-utils/utils.js

```diff
@@ -174,11 +180,11 @@ function generateDownloadCliErrorMessage(downloadUrl, cliVersion) {
     let errMsg = 'Failed while attempting to download JFrog CLI from ' + downloadUrl + '. ';
     if (downloadUrl === buildReleasesDownloadUrl(cliVersion)) {
         errMsg +=
-            "If this build agent cannot access the internet, you may use the 'Artifactory Tools Installer' task, to download JFrog CLI through an Artifactory repository, which proxies " +
+            "\nIf this build agent cannot access the internet, you may use the 'Artifactory Tools Installer' task, to download JFrog CLI through an Artifactory repository, which proxies " +
             buildReleasesDownloadUrl(cliVersion) +
```

Comment:
The proxy URL is: `https://releases.jfrog.io/artifactory/jfrog-cli/v2-jf`
```suggestion
            jfrogCliReleasesUrl +
```